### ISSUE TITLE
Nothing on first main screen in app, only the Account sign and the 3 buttons at down.

### ISSUE THREAD

**Issue by AttilaSari** [owncloud/ios-app#786]
### Steps to reproduce
1. Main screen is nearly empty. Only the 3 buttons are exist on down. 
2. After restart the error is still exists. even phone restart neither helped.
Other clients work fine (Windows PC) and from web-browser.

### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### Client
**iOS version: 13.6.1

**ownCloud app version: the last version from App Store. Maybe 11.4

**Device model: iPhone Pro 64GB

### Server configuration
**Web server: apache2

**Database: mariadb

**PHP version:7.3

**ownCloud version: 10.5.0

**Storage backend (external storage): external hdd

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

**Comment by hosy** [comment]
@AttilaSari van you attach a screenshot please?

**Comment by cdamken** [comment]
I had the same behavior:

https://cloud.damken.com/index.php/s/b9EENBuAxHEi2Zu

I tried to upload  it but is bigger than 10MB (Sorry @hosy )

@AttilaSari As a workaround, I saw that leaving the app running about 3 minutes was enough to starting to be responsive again.
My explanation (@hosy should correct me if I'm wrong): I have over 1 Million files, most of them are in external Storages (Google, Dropbox, WindowsNetwork Drive and SMB)  and it took a time to update the root folders (I suppose). After that, it was a little slow the next day but now it works fine.

**Comment by AttilaSari** [comment]
> @AttilaSari van you attach a screenshot please?

i can't attach, because i need a quick solution... so i delete the app and reconfig everything. and that was solved the problem.

**Comment by hosy** [comment]
@cdamken thank you for sharing. This is only after the upgrade to the new version? We should used chached data and reload it in background with an `Refreshing…` label on the bottom @felix-schwarz

**Comment by felix-schwarz** [comment]
@hosy The client UI only gets fully set up after the `OCCore` has started up, so that commands triggered by UI interactions have a destination.

Usually, the core should be up and running so quick that one would not typically notice the delay.

Situations where I could imagine it takes longer would be when there's housekeeping to do at launch - like upgrading the database of the account to a newer scheme, or cleaning up storage space.

Scanning for changes, on the other hand, should not impact core startup time at all, as it only starts once the core is up and running.

Maybe we should add a placeholder `Starting up…` progress indicator to indicate the connection is in fact being opened?

**Comment by jesmrec** [comment]
Not able to reproduce the problem after installing from scratch and upgrading from latest version with a pretty populated account.

**Comment by jesmrec** [comment]
I gave a last try by using the `Low Data Mode` to simulate bad connection. I could not reproduce either. 

Is it OK to move forward and getting more feedback from final users?

**Comment by michaelstingl** [comment]
> I gave a last try by using the `Low Data Mode` to simulate bad connection. I could not reproduce either.

No, not `Low Data Mode`, enable Network Link Conditioner:
https://nshipster.com/network-link-conditioner/#enabling-network-link-conditioner-on-ios-devices

**Comment by jesmrec** [comment]
Thanks, i did not use that tool before. 

I have performed a couple of tests with the `Very Bad Network`, `Edge` and `100% Loss` and i see the lagged responses but not the specified behaviour.

**Comment by hosy** [comment]
Does it make sense to include a `Starting up…` label and a `Cancel` button where the user can cancel the view and go back to the account overview?

**Comment by hosy** [comment]
I would implement it like this:

![Simulator Screen Shot - iPhone 11 Pro - 2020-09-22 at 11 13 56](https://user-images.githubusercontent.com/736109/93864180-ca893280-fcc4-11ea-9ac8-c8e542b386b7.png)

**Comment by felix-schwarz** [comment]
Fixed by #801.
